test/test_distributions.py

```diff
@@ -1878,9 +2062,12 @@ def test_dirichlet_shape(self):
         self.assertEqual(dist._event_shape, torch.Size((2,)))
         self.assertEqual(dist.sample().size(), torch.Size((3, 2)))
         self.assertEqual(dist.sample((5, 4)).size(), torch.Size((5, 4, 3, 2)))
-        self.assertEqual(dist.log_prob(self.tensor_sample_1).size(), torch.Size((3,)))
+        simplex_sample = self.tensor_sample_1 / self.tensor_sample_1.sum(-1).unsqueeze(-1)
+        self.assertEqual(dist.log_prob(simplex_sample).size(), torch.Size((3,)))
         self.assertRaises(ValueError, dist.log_prob, self.tensor_sample_2)
-        self.assertEqual(dist.log_prob(Variable(torch.ones((3, 1, 2)))).size(), torch.Size((3, 3)))
+        simplex_sample = Variable(torch.ones((3, 1, 2)))
```

Comment:
I think this is simply `torch.ones(...)` now. You should remove all references to `Variable`.